setup.py

```diff
@@ -1,8 +1,5 @@
-from pathlib import Path
 from setuptools import setup, find_packages
-
-this_directory = Path(__file__).parent
-version = (this_directory / "VERSION").read_text().strip()
```

Comment:
was this no bueno?

Reply:
I really liked the VERSION file concept more so than the version.py concept

Reply:
This didn't allow us to use the version number within the module. So we still had to have the version number in two places. I looked at how `mypy` does version numbers and copied them for the most part

Reply:
gotcha ok